man/fi_mr.3.md

```diff
@@ -941,6 +941,14 @@ The follow flag may be specified to any memory registration call.
   region. When set, the base MR is specified through the base_mr field of the
   fi_mr_attr structure. This flag cannot be used in conjunction with FI_MR_DMABUF.
 
+*FI_MR_SINGLE_USE*
```

Comment:
We already have FI_MR_DMABUF which is a flag instead of a mode, and there are other MR flags not starting with FI_MR_ at all. Unless we want to systematically rename various flags etc, I'll just leave it as it is for now.